src/templates/base/base.html

```diff
@@ -275,7 +275,7 @@
       <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}#ebook">{{ self.ebook_title() }}</a>
     </li>
     <li class="nav-dropdown-list-chapter ebook">
-      <a href="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf">{{ self.ebook_download_short() }}</a>
+      <a class="ebook-menu-download" href="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf">{{ self.ebook_download_short() }}</a>
```

Comment:
```suggestion
      <a class="ebook-menu-download" href="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf" data-event="ebook-click">{{ self.ebook_download_short() }}</a>
```

Reply:
Good idea. Based it on the link one too much instead of thinking how to do this properly.